### PR TITLE
[Website] Add coauthor option to case-studies

### PR DESCRIPTION
**Pull request by bullet03**
Rsolves #24486
- update `case-studies/list.html`, add options to case-studies to be able to add сoauthor to `https://beam.apache.org/case-studies/`
- update related ADD_CASE_STUDY.md where it is noted how to add a new case-study
------------------------

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

File: website/ADD_CASE_STUDY.md [hunk 1]
```diff
@@ -27,18 +27,21 @@ limitations under the License.
...
+| `name`                | Company or project name                                                                                                        |
+| `icon`                | Relative path to the company/project logo e.g. "/images/logos/powered-by/company_name.png"                                     |
+| `category`            | `study` for case studies                                                                                                       |
+| `cardTitle`           | Case study card title for Apache Beam [case studies](https://beam.apache.org/case-studies/) page                               |
+| `cardDescription`     | Description for [case studies](https://beam.apache.org/case-studies/) page, usually 30-40 words                                |
+| `authorName`          | Case study author                                                                                                              |
+| `secondAuthorName`    | Case study additional author, optional param                                                                                   |
```
Comment (AKosolapov) [inline website/ADD_CASE_STUDY.md:39]: Consider changing the parameter name to coauthrorName?

File: website/ADD_CASE_STUDY.md [hunk 2]
```diff
@@ -27,18 +27,21 @@ limitations under the License.
...
+| `authorPosition`      | Case study author role                                                                                                         |
+| `secondAuthorPosition`| Case study additional author role, optional param                                                                              |
```
Comment (AKosolapov) [inline website/ADD_CASE_STUDY.md:41]: Similarly - Consider changing the parameter name to coauthrorPosition?

File: website/ADD_CASE_STUDY.md (hunk 1 above)
Comment (bullet03) [reply website/ADD_CASE_STUDY.md]: changed

File: website/ADD_CASE_STUDY.md (hunk 2 above)
Comment (bullet03) [reply website/ADD_CASE_STUDY.md]: changed

**Comment by AKosolapov** [comment]
LGTM

**Comment by bullet03** [comment]
R: @pabloem, could you kindly help with review?

**Comment by pabloem** [comment]
nice! LGTM

**Comment by bullet03** [comment]
> nice! LGTM

Thank you!

**Pull request merged**